Systems/c172p-hydrodynamics.xml

```diff
@@ -274,11 +274,14 @@
        </min>
       </max>
      </product>
+     <!-- Squat is disabled as it currently missbehaves in near at rest
```

Comment:
It seemed Github had no problems with the rebase, so can you fix the "missbehaves" typo? :smile: